typescript/packages/plugins/balancer/src/balancer.service.ts

```diff
@@ -0,0 +1,189 @@
+import {
+    AddLiquidity,
+    AddLiquidityInput,
+    AddLiquidityKind,
+    BalancerApi,
+    ChainId,
+    ExactInQueryOutput,
+    InputAmount,
+    RemoveLiquidity,
+    RemoveLiquidityInput,
+    RemoveLiquidityKind,
+    Slippage,
+    Swap,
+    SwapBuildOutputExactIn,
+    SwapKind,
+    Token,
+    TokenAmount,
+} from "@balancer/sdk";
+import { Tool } from "@goat-sdk/core";
+import { EVMWalletClient } from "@goat-sdk/wallet-evm";
+import { LiquidityParameters, RemoveLiquidityParameters, SwapParameters } from "./parameters";
+
+export class BalancerService {
+    private getBalancerApi(chainId: ChainId) {
+        console.log("chainId from getbalancerapi", chainId);
+        return new BalancerApi("https://api-v3.balancer.fi/", ChainId.POLYGON);
+    }
+
+    private rpcUrl = "https://polygon.llamarpc.com";
```

Comment:
I would make this a config parameter for the plugin

Reply:
done